.gitignore

```diff
@@ -1,2 +1,2 @@
-git/gitconfig.local.symlink
+developer/common/git/gitconfig.local.symlink
```

Comment:
We can improve this by entering `**/*.symlink` in the `.gitignore`. This makes it more flexible.

Reply:
Does this also gitignore the `gitconfig.symlink` and `gitignore.symlink` files?

Not sure why we would want to that.